src/Components/ServiceScene/Breakdowns/LayoutSwitcher.tsx

```diff
@@ -15,15 +16,7 @@ export interface LayoutSwitcherState extends SceneObjectState {
 export type LayoutType = 'single' | 'grid' | 'rows';
 
 export class LayoutSwitcher extends SceneObjectBase<LayoutSwitcherState> {
-  public Selector({ model }: { model: LayoutSwitcher }) {
-    const { active, options } = model.useState();
-
-    return (
-      <Field>
-        <RadioButtonGroup options={options} value={active} onChange={model.onLayoutChange} />
-      </Field>
-    );
-  }
+  public static Selector = LayoutSwitcherComponent;
```

Comment:
```suggestion
  public static Component = LayoutSwitcherComponent;
```
Does it make sense for it to be called `Selector`?

Reply:
LayoutSwitcher already has a `Component`, would `LayoutSelector` be better?

Reply:
Nah, only valid if there was no Component property.